lib/diplomat.rb

```diff
@@ -21,7 +21,7 @@ def require_libs(*libs)
   self.root_path = File.expand_path "..", __FILE__
   self.lib_path = File.expand_path "../diplomat", __FILE__
 
-  require_libs "configuration", "rest_client", "kv", "datacenter", "service", "members", "nodes", "check", "health", "session", "lock", "error", "event"
+  require_libs "configuration", "rest_client", "kv", "datacenter", "service", "members", "node", "nodes", "check", "health", "session", "lock", "error", "event"
```

Comment:
This line is getting a little long and now seems like a good time to wrap it into two or more. Would you mind?